packages/core/src/utilities/historyMemo/index.ts

```diff
@@ -64,6 +64,7 @@ export class HistoryMemo {
    */
   public undo(items = 1) {
     while (items > 0 && this.undoAvailable > 0) {
+      console.log('running udno');
```

Comment:
Can you start using the log framework?  Having debug level logs for this sort of thing can be quite useful to be able to turn on to debug things.